galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,161 @@
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+from pulp_ansible.app.models import (
+    CollectionVersion,
+    AnsibleDistribution
+)
+from galaxy_ng.app.models import Namespace
+from django.conf import settings
+from galaxy_ng.app.api import base as api_base
+
+golden_name = settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH
```

Comment:
move this into `get`. You're not using it in any other methods, so it's best not to pollute the global namespace.